.github/workflows/static-analysis-test.yaml

```diff
@@ -0,0 +1,33 @@
+name: Static analysis test
+on:
+  pull_request:
+    branches:
+      - master
+    # These hardcoded paths are set for the PoC on the Apache chart
+    paths:
+      - "bitnami/apache/templates/*"
+      - "bitnami/apache/values.yaml"
```

Comment:
How do you plan to solve the problem of running the "Static analysis test" only for the charts that a specific PR modifies?

Reply:
We'll have to do something similar to the action for readme-generator: https://github.com/bitnami/charts/blob/master/.github/workflows/generate-chart-readme.yml#L41-L51
TBH it's a hack for a functionality that should be provided by Github and be accesible via something like
`${{ github.event.paths }}`
We can go with it or explore other options like https://github.com/dorny/paths-filter

Reply:
Well.. At lease we have some workarounds for this problem